using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +148,60 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+By default, a continuous aggregate is populated with data when it is created. 
+means that it is immediately populated with data. Since this can be a
```

Comment:
```suggestion
By default, a continuous aggregate is populated with data when it is created, which
means that it is immediately populated with data. Since this can be a
```

Reply:
If you choose to adopt my suggestion above, this paragraph can probably be removed.